rfc/text/advanced/ap_session_meta_api.md

```diff
@@ -39,11 +39,15 @@ Fired when a session leaves a realm on the router or is disconnected. The event
 
 #### Session Meta Procedures
 
-A client can actively retrieve information about sessions via the following meta-procedures:
+A client can actively retrieve information about sessions, or forcefully disconnect sessions, via the following meta-procedures:
```

Comment:
Should say "forcefully **close** sessions", instead of disconnect. The spec uses the terms "connect"/"disconnect" within the context of transports.